### ISSUE TITLE
[Feature Request] Allow the combination of Monochrome + Outline on text

### ISSUE THREAD

**Issue by Koviko** [Tercioo/Plater-Nameplates#84]
I'd like to be able to use Monochrome + Outline together. [This option is standard in WeakAuras as well.](https://user-images.githubusercontent.com/835767/107987967-05a24000-6f9d-11eb-949c-caa73fa3fd65.png)

The addition is fairly simple:

[Plater_OptionsPanel.lua:872-873](
https://github.com/Tercioo/Plater-Nameplates/blob/510ceef223255c9580b8ada9946daba548749961/Plater_OptionsPanel.lua#L872-L873):

```diff
-	local outline_modes = {"NONE", "MONOCHROME", "OUTLINE", "THICKOUTLINE"}
+	local outline_modes = {"NONE", "MONOCHROME", "OUTLINE", "THICKOUTLINE", "MONOCHROME, OUTLINE"}
-	local outline_modes_names = {"None", "Monochrome", "Outline", "Thick Outline"}
+	local outline_modes_names = {"None", "Monochrome", "Outline", "Thick Outline", "Monochrome Outline"}
```

From what I can tell in a day of testing, this addition doesn't seem to have any side effects or throw any errors.

**Comment by cont1nuity** [comment]
Sure. Do you want to create a PR for this? If so: adding the "Monochrome Thick Outline" option makes sense as well.

**Comment by Koviko** [comment]
#85

**Comment by cont1nuity** [comment]
Closed via #86
